packages/forms/src/styled/radio/StyledRadioLabel.ts

```diff
@@ -40,6 +40,8 @@ export const StyledRadioLabel = styled(StyledLabel).attrs({
   position: relative;
   cursor: pointer;
   user-select: none;
+  font-weight: ${props =>
+    props.isRegular === false ? props.theme.fontWeights.semibold : props.theme.fontWeights.regular};
```

Comment:
I had misunderstood https://github.com/zendeskgarden/react-components/pull/1076#discussion_r610846183. This has been reverted since it exists in `StyledLabel`.